app/styles/crate.scss

```diff
@@ -313,6 +313,13 @@
         pre {
             overflow-x: auto;
         }
+
+        p {
+            code {
+                background-color: #fff;
```

Comment:
What? It's rendered perfectly for me and the text is readable as my screenshot shows it. The point is to make text code blocks stand out a bit more.